src/app/work-item/work-item-list/work-item-list.component.html

```diff
@@ -1,33 +1,33 @@
 <app-header></app-header>
 <div class="container-fluid screen-min-ht">
 	<div class="list-group list-view-pf list-view-pf-view">
-		 <div>
+		 <div id="workItemList_quickAdd">
 			 <div class="list-group-item supress-hover" (click)="addWorkItem()">
 				<work-item-quick-add (close)="close($event)"></work-item-quick-add>
 			</div>
 		</div>
-		<div *ngFor="let workItem of workItems">
-			<div class="list-group-item" (click)="onSelect(workItem)"
+		<div *ngFor="let workItem of workItems" id="workItemList_OuterWrap">
+			<div class="list-group-item" id="workItemList_Container" (click)="onSelect(workItem)"
```

Comment:
for workItemList_Container if might not need this after the final designs as we  can figure out the exact outer wrapper for this